src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -150,7 +149,7 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 			providers = [...this._providers.values()].flatMap(providerMap => [...providerMap.values()]);
 		}
 
-		if (!extensionCompletionsEnabled || skipExtensionCompletions) {
+		if (skipExtensionCompletions) {
```

Comment:
before, for the tests, we were passing in `false` for extension completion enablement, so it would go in here. without this, it doesn't enter this, so it fails.

Reply:
fixed it, the id I was passing in for the test provider was wrong, so our filtering was not working